pkg/apply/applier.go

```diff
@@ -188,6 +185,65 @@ func splitInfos(infos []*resource.Info) ([]*resource.Info, []*resource.Info) {
 	return resources, groupingObjectTemplates
 }
 
+// buildTaskQueue takes the slice of infos and object identifiers, and
+// builds a queue of tasks that needs to be executed.
+func (a *Applier) buildTaskQueue(infos []*resource.Info, identifiers []object.ObjMetadata, eventChannel chan event.Event) chan taskrunner.Task {
+	tasks := []taskrunner.Task{
+		&task.ApplyTask{
+			Objects:      infos,
+			ApplyOptions: a.ApplyOptions,
+		},
+		&task.SendEventTask{
```

Comment:
It's not clear why we're addling the `SendEventTask` at the same time as the `ApplyTask`; maybe a comment?

Reply:
I have added comments to all the tasks added to the queue.